pkgs/development/tools/build-managers/bob/default.nix

```diff
@@ -17,6 +17,14 @@ buildGoModule rec {
 
   excludedPackages = [ "example/server-db" "test/e2e" "tui-example" ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
```

Comment:
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Reply:
thansk for the suggestion. I didn't know how to safely wrap the script for cross-compile.